containers/daap-create-metadata/tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+boto3==1.28.40
```

Comment:
in the other lambdas I've been including the source requirements here, i.e. starting the file with

`-r ../src/var/task/requirements.txt`

and then just adding the test requirements.

I don't know if at some point we want to switch over to poetry or something else that lets us define dev dependencies vs normal dependencies in one place?